src/lib/pipeline.ml

```diff
@@ -164,13 +168,20 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
   open Option
   module Stdlib = Biokepi.EDSL.Library.Make(Bfx)
 
-  let to_bam ~parameters ~reference_build input =
-    let list_of_inputs = Stdlib.bwa_mem_opt_inputs input in
-    List.map list_of_inputs ~f:(Bfx.bwa_mem_opt ~reference_build ?configuration:None)
+
+  let to_bam ~parameters ~reference_build samples =
```

Comment:
Does this function only get called for WES data? (since you're running BWA-mem)

Will you remind me why we're not running indel realigner?